src/graph-model.js

```diff
@@ -1,5 +1,23 @@
 var randomColorRGB = require('random-color-rgb'),
-shell = require('shelljs');
+    shell          = require('shelljs'),
+    compose        = require('./compose');
+
+function build(config, cucumberJSON) {
+    return Promise
+        .all(coverageData(config, cucumberJSON))
+        .then(compose(model, transform, flatten));
```

Comment:
@cookiescrumbs this is where I'm composing the 3 functions into 1 function that calls the 3 in reverse order, I'm wondering whether to change the compose function so that it calls them in the actual order, so it's more like the elixir pipeline syntax (does elm have that syntax too?) so flatten |> transform |> model. Yeah, I'll change that compose function that I pinched from someone's blog post to work the other way around :-)

Reply:
is it possible to do this? 🤔 

```
.then(flatten)
.then(transform)
.then(model)
```

Reply:
yes, you can!